src/helics/core/fileConnections.hpp

```diff
@@ -53,14 +54,13 @@ void makeConnectionsToml(brkX* brk, const std::string& file)
             }
         }
     }
-    auto filts = doc.find("filters");
-    if (filts != nullptr) {
-        auto& filtArray = filts->as<toml::Array>();
+    auto filts = toml::find_or(doc, "filters", uVal);
+    if (!filts.is_uninitialized()) {
+        auto& filtArray = filts.as_array();
```

Comment:
Same as most of the others -- looks like another place that might need an is_array check.